### PR TITLE
Added calico test for E2E

### PR DESCRIPTION
**Pull request by jayanthvn**
*Issue #, if available:*

*Description of changes: Added Calico test for weekly test runs

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -252,7 +282,7 @@ workflows:
           requires:
             - build_x86_64
             - build_aarch64
-      - integration_test:
+      - calico_test:
```
Comment (mogren) [inline .circleci/config.yml:285]: We probably still want the regular `integration_test` here.

File: .circleci/config.yml [hunk 2]
```diff
@@ -303,3 +333,4 @@ workflows:
       - bottlerocket_test:
           requires:
             - warm_eni_test
+      - calico_test
```
Comment (mogren) [inline .circleci/config.yml:336]: Can we put this first, before the `performance_test`?

File: scripts/run-integration-tests.sh [hunk 3]
```diff
@@ -196,6 +197,17 @@ if [[ $RUN_KOPS_TEST == true ]]; then
...
+if [[ $RUN_CALICO_TEST == true ]]; then
+    $KUBECTL_PATH apply -f "$TEST_CALICO_PATH"
+    sleep 5
+    while [[ $($KUBECTL_PATH describe ds calico-node -n=kube-system | grep "Available Pods: 0") ]]
+    do
+        sleep 5
+        echo "Waiting for calico daemonset update"
+    done
+    echo "Updated calico daemonset!"
+fi
```
Comment (mogren) [inline scripts/run-integration-tests.sh:209]: This just applies the Calico config. We should probably add some network policy as well, and have some custom test verifying that the policy has been applied.

File: .circleci/config.yml (hunk 1 above)
Comment (jayanthvn) [reply .circleci/config.yml:285]: Yeah this is just test on my local account, should have probably marked it as draft :D..will revert it soon.

**Comment by jayanthvn** [comment]
Have another opened another PR so closing this.

**Pull request closed without merging**